### ISSUE TITLE
git tags

### ISSUE THREAD

**Issue by doctaweeks** [facebook/wdt#132]
It would be very nice to have git tags created for each version as its bumped by the bot.

**Comment by ldemailly** [comment]
We manually make periodic [releases](https://github.com/facebook/wdt/releases) that are tagged (using the same scheme, e.g. https://github.com/facebook/wdt/tree/v1.26.1603040) with explanations/summary of the main changes - though I haven't done one in a while

out of curiosity what would you do with the git tags ?

not sure how to set that up though - the way the version is updated automatically happens inside fb's source tree and then get synced here

**Comment by doctaweeks** [comment]
It's a lot easier to reference code history beginning from a version string if there is a matching tag rather than tracking down a commit ID responsible for the changing the version.

Personally, I have a "watcher" for many projects on GitHub and perform updates as I get notifications of new tags. However, since March, updating wdt has been a manual process - linking a version string to commit ID and treating the hash as the "version".

FWIW, folly has a similar date-based version scheme and ~2 months ago @bhamiltoncx reintroduced tagging.

**Comment by ldemailly** [comment]
@fredemmott can shipit do that ?

**Comment by fredemmott** [comment]
No, you want a different internal project (see Shawn's response on your internal copy of this question)

**Comment by ldemailly** [comment]
@doctaweeks I just made a new manual tag but if you would use anything as a version, please use the actual version that gets automatically updated after each commit that passes our internal tests, an example of such commit is:

https://github.com/facebook/wdt/commit/6fe814083e5e1583da4e3ee69c0ed16fdef200fc

it's always coming from svcscm@fb.com and changes CMakeLists.txt / WdtConfig.h (cmake creates the WdtConfig.h used for open source builds from https://github.com/facebook/wdt/blob/master/WdtConfig.h.in
